src/dropin.js

```diff
@@ -226,17 +247,18 @@ Dropin.prototype = Object.create(EventEmitter.prototype, {
 
 Dropin.prototype._initialize = function (callback) {
   var localizedStrings, localizedHTML, paypalScriptOptions;
-  var dropinInstance = this; // eslint-disable-line consistent-this
-  var container = this._merchantConfiguration.container || this._merchantConfiguration.selector;
+  var self = this; // eslint-disable-line consistent-self
+  var container = self._merchantConfiguration.container || self._merchantConfiguration.selector;
```

Comment:
I feel like we switch out `this` for `self` in a lot in this function. Is that required? I'd prefer to use `this` in most cases unless we have some crazy nested anonymous functions

Reply:
I switched to using `self` everywhere in this function because there are my functions where we would have to bind(this) and it seemed cleaner to just use self everywhere. If you feel strongly about using this and bind for this function, we can, but I preferred going the self route.

Reply:
This was more of a nitpick. `this` stands out as a keyword better than `self` but otherwise I'm OK with using `self`.